exercises/exercise3.js

```diff
@@ -1,12 +1,444 @@
+          ", " +
+          `${
+            answer2.charAt(0).toUpperCase() + answer2.slice(1).toLowerCase()
+          }` +
+          " is the correct answer. Well done! Your score is " + (state.score += 10) + "." + "\nNext question: What is the longest river in Africa?"
+        );
+      } else {
+        return "Sorry, that's wrong. All that you are you owe to your parents. Why don't you send them a penny and square the account?";
+      }
+    },
+    isListening: true,
+  },
+  {
+    label: "Question 3",
+    prompt: "Q3: #{answer3}",
+    handler: function (params, state) {
+      const answer3 = params.answer3;
+      if (answer3.toLowerCase() === "nile") {
+        return (
+          "Nice job " +
+          state.firstName +
+          ", " +
+          "the " +
+          `${
+            answer3.charAt(0).toUpperCase() + answer3.slice(1).toLowerCase()
+          }` +
+          " is the correct answer. Well done! Your score is " + (state.score += 10) + "." + "\nNext question: Mumbai, Chennai, and Kolkata are metropolitan cities in which country?"
+        );
+      } else {
+        return "Sorry, that's wrong. Any similarity between you and a human is purely coincidental!";
+      }
+    },
+    isListening: true,
+  },
+  {
+    label: "Question 4",
+    prompt: "Q4: #{answer4}",
+    handler: function (params, state) {
+      const answer4 = params.answer4;
+      if (answer4.toLowerCase() === "india") {
+        return (
+          "Nice job " +
+          state.firstName +
+          ", " +
+          `${
+            answer4.charAt(0).toUpperCase() + answer4.slice(1).toLowerCase()
+          }` +
+          " is the correct answer. Well done! Your score is " + (state.score += 10) + "." + "\nNext question: What country calls itself Nippon?"
+        );
+      } else {
+        return "Sorry, that's wrong. Are your parents siblings?";
+      }
+    },
+    isListening: true,
+  },
+  {
+    label: "Question 5",
+    prompt: "Q5: #{answer5}",
+    handler: function (params, state) {
+      const answer5 = params.answer5;
+      if (answer5.toLowerCase() === "japan") {
+        return (
+          "Nice job " +
+          state.firstName +
+          ", " +
+          `${
+            answer5.charAt(0).toUpperCase() + answer5.slice(1).toLowerCase()
+          }` +
+          " is the correct answer. Well done! Your score is " + (state.score += 10) + "." + "\nNext question: What is the largest non-continental island in the world?"
+        );
+      } else {
+        return "Sorry, that's wrong. Brains aren't everything. In fact, in your case they're nothing!";
+      }
+    },
+    isListening: true,
+  },
+  {
+    label: "Question 6",
+    prompt: "Q6: #{answer6}",
+    handler: function (params, state) {
+      const answer6 = params.answer6;
+      if (answer6.toLowerCase() === "greenland") {
+        return (
+          "Nice job " +
+          state.firstName +
+          ", " +
+          "the " +
+          answer6 +
```

Comment:
Random, but you didn't use your fancy .charAt().toUpperCase() etc on this one. Still works and is a perfectly acceptable function, just wondered if it was missed.